jpa/spec-tests/src/main/java/ee/jakarta/tck/persistence/core/types/datetime/DateTimeEntity.java

```diff
@@ -0,0 +1,306 @@
+/*
+ * Copyright (c) 2017, 2024 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+package ee.jakarta.tck.persistence.core.types.datetime;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.OffsetDateTime;
+import java.time.OffsetTime;
+import java.time.Year;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.NamedQuery;
+import jakarta.persistence.Table;
+
+/**
+ * Entity containing new JPA date and time types.
+ */
+@Entity
+@Table(name = "DATATYPES_DATE_TIME")
```

Comment:
Are we automatically creating the DATATYPES_DATE_TIME table somewhere?

Reply:
I guess so via https://github.com/jakartaee/platform-tck/pull/1240/files#diff-e4a21611b5e0c0ce1eceb4929a0fcf820d562e6fc17ef810fa168a9def899cfcR69 :)

Reply:
> I guess so via https://github.com/jakartaee/platform-tck/pull/1240/files#diff-e4a21611b5e0c0ce1eceb4929a0fcf820d562e6fc17ef810fa168a9def899cfcR69 :)

Yes, that is right persistence property.